### PR TITLE
Wait for window to load before installing ServiceWorker.

### PR DESCRIPTION
**Pull request by cramforce**
The WPT waterfalls in #5573 showed that the SW can interfere pretty heavily with the current page load which is bad in cases where it downloads a lot of stuff not currently needed.

### CODE REVIEW

**Comment by jridgewell** [comment]
> which is bad in cases where it downloads a lot of stuff not currently needed

Ours doesn't download anything. This is already an async and can never affect _this_ page load (it won't start until the next navigation).

**Comment by cramforce** [comment]
I wasn't referring to our SW. See this waterfall and how the SW starts downloading resources while the current page is still loading
https://www.webpagetest.org/waterfall.php?test=161012_SM_3E8Y&run=1&cached=0&step=1&max=11&mime=1&state=1&cpu=1&bw=1

**Comment by jridgewell** [comment]
Ha, I keep seeing Service Worker and think "Cache Service Worker".

File: extensions/amp-install-serviceworker/0.1/amp-install-serviceworker.js [hunk 1]
```diff
@@ -69,7 +68,9 @@ export class AmpInstallServiceWorker extends AMP.BaseElement {
     }
 
     if (parseUrl(win.location.href).origin == parseUrl(src).origin) {
-      install(this.win, src);
+      this.laodPromise(() => {
```
Comment (jridgewell) [inline extensions/amp-install-serviceworker/0.1/amp-install-serviceworker.js]: "loadPromise"

**Pull request merged**